heat/core/sanitation.py

```diff
@@ -27,6 +28,120 @@
 ]
 
 
+def sanitize_distribution(
+    *args: DNDarray, target: DNDarray, diff_map: torch.Tensor = None
```

Comment:
It should be `*args: Sequence[DNDarray, ...]` here. Unless we need to fix the number of DNDarrays or the type of sequence.